### DISCUSSION TITLE
Support response streaming in Plugins mode

### DISCUSSION THREAD

**Discussion by tamikomssi** [danny-avila/LibreChat#1451] [Feature Requests & Suggestions]
It would be much more convenient for the user's point of view to see the response generated immediately instead of waiting 1-2 min before seeing the answer. Moreover this would enable the use of gpt-4-1106-preview which is currently extremely slow compared to gpt-4 in plugins mode.

**Comment by danny-avila** [comment]
Thanks, this is something I want as well. The future of plugins/tools will see an overhaul soon where this will be revisited

**Reply by gloriousknight** [reply]
Hi, I would like to know in which version this feature will be added?

**Reply by berry-13** [reply]
https://github.com/danny-avila/LibreChat/discussions/2041#discussioncomment-8730461

**Reply by gloriousknight** [reply]
Thanks.
